nixos/modules/services/networking/cloudflared.nix

```diff
@@ -302,7 +302,7 @@ in
             mkConfigFile = pkgs.writeText "cloudflared.yml" (builtins.toJSON fullConfig);
           in
           nameValuePair "cloudflared-tunnel-${name}" ({
-            after = [ "network.target" ];
+            after = [ "network-online.target" ];
```

Comment:
I'm reading https://www.freedesktop.org/software/systemd/man/systemd.special.html and wonder whether network.target should not be removed here (just append network-online.target). Thoughts?

Reply:
The idea being that After=network.target will ensure proper shutdown of the service when the system gets powered off. I'm not sure what network-online.target does there. I'm guessing NOP?

Reply:
Yes, with only network-online.target the service doesn't get shutdown cleanly. So conserving network.target and adding network-online.target is the way to go.